### ISSUE TITLE
SARIMAX: scaling exogenous regressors and forecast value

### ISSUE THREAD

**Issue by pisarevaa** [statsmodels/statsmodels#7382]
Hi, everone! First question, should I scale exogenous regressors before using in fitting SARIMAX model or not? Second question, should I scale exogenous regressors and also target/forecasting value to the same scale before using in fitting SARIMAX model or not? I'm using sklearn.preprocessing.StandardScaler library fo scaling.

**Comment by bashtage** [comment]
You usually don't need to scale the endog variable since it is automatically self-normalizing.  You may want to rescale exog so that their expected coefficients are similar in magnitude to the coefficients in the ARMA part of the model.  This will enhance numerically stability of the solution.

**Comment by pisarevaa** [comment]
Thanks a lot!
